File: app/src/main/java/com/kose/milkytea/ui/LauncherActivity.java
package com.kose.milkytea.ui;

import android.content.Intent;
import android.os.Handler;

import com.kose.milkytea.R;
import com.kose.milkytea.databinding.ActivityLauncherBinding;
import com.kose.milkytea.ui.base.BaseActivity;
import com.kose.milkytea.ui.home.MainActivity;

public class LauncherActivity extends BaseActivity<ActivityLauncherBinding> {

    @Override
    protected void initView() {
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent();
                intent.setClass(LauncherActivity.this, MainActivity.class);
                startActivity(intent);
                finish();
            }
        }, 3000);
    }

    @Override
    protected int getLayoutId() {
        return R.layout.activity_launcher;
    }
}

File: app/src/main/java/com/kose/milkytea/LauncherActivity.kt
package com.kose.milkytea


import android.os.Handler
import com.kose.common.base.ui.BaseActivity
import com.kose.milkytea.databinding.ActivityLauncherBinding
import com.kose.router.IRouter
import com.kose.router.KRouter

/**
 * @Description
 * @Author KoseGuo
 * @Since 2023/4/5 15:20
 */
class LauncherActivity : BaseActivity<ActivityLauncherBinding>() {

    override fun initView() {
        Handler().postDelayed({
            KRouter.instance.startActivity(IRouter.HOMEPAGE)
        }, 3000)
    }

    override fun getLayoutId(): Int {
        return R.layout.activity_launcher
    }
}